### PR TITLE
fix: remove install instructions in README

### PR DESCRIPTION
**Pull request by srodenhuis**
APL can only be used in combination with a DNS zone. The use of `minimal-values` combined with `nip.io` will no longer be supported. This PR removes the install instructions from the README and now links to the installation instructions in https://apl-docs.net. The docs are updated in https://github.com/linode/apl-docs/pull/80

### CODE REVIEW

**Review by merll — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -20,35 +20,42 @@
...
+**Developers** - With easy self-service
+
+- Build OCI compliant images from source code
+- Deploy containerized workloads the GitOps way using the provided quickstarts or BYO golden path templates
+- Automatically update container images of workloads
+- Publicly expose applications
+- Get instant access to logs, metrics and traces
+- Store images in a private registry
+- Configure network policies, response headers and CNAMEs
+- Check applications against a comprehensive set of build-in security policies
```
Comment (CasLubbers) [inline README.md:43]:
```suggestion
- Check applications against a comprehensive set of built-in security policies
```

File: README.md [hunk 2]
```diff
@@ -20,35 +20,42 @@
...
+- Create and manage secrets
+- Create private Git repositories and use the build-in CI/CD pipelines
```
Comment (CasLubbers) [inline README.md:45]:
```suggestion
- Create private Git repositories and use the built-in CI/CD pipelines
```

File: README.md [hunk 3]
```diff
@@ -20,35 +20,42 @@
...
+- Get all the required capabilities in a pre-configured, integrated and automated way
+- Onboard development Teams in a comprehensive multi-tenant setup and make them self-serving
+- Manage users
+- Ensure governance with security policies
+- Implement zero-trust networking
+- Change the desired state of the platform based on Configuration-as-Code
+- Support multi- and hybrid cloud scenarios
+- Prevent cloud provider lock-in
+- Implement full observability
+- Comply to Disaster Recovery requirements
```
Comment (CasLubbers) [inline README.md:58]:
```suggestion
- Comply with Disaster Recovery requirements
```

**Review by CasLubbers — APPROVED** [review]
Some small remarks

**Pull request merged**